var/spack/repos/builtin/packages/htslib/package.py

```diff
@@ -52,6 +52,10 @@ class Htslib(AutotoolsPackage):
     depends_on("automake", when="@1.2")
     depends_on("libtool", when="@1.2")
 
+    @property
+    def libs(self):
+        return find_libraries("*", root=self.prefix, recursive=True)
```

Comment:
You should include the exact name of the library. Otherwise, if it's installed in `/usr/lib`, you'll pick up thousands of libraries.

Reply:
I think it's `lilbhts`? This is the output of `find_libraries("*", root=self.prefix, recursive=True)` :

```
SPACK_DIR/htslib-1.16-su7mfvegt5xjb7o7lzulziwcigvpna64/lib/libhts.1.16.dylib
SPACK_DIR/htslib-1.16-su7mfvegt5xjb7o7lzulziwcigvpna64/lib/libhts.dylib
SPACK_DIR/htslib-1.16-su7mfvegt5xjb7o7lzulziwcigvpna64/lib/libhts.3.dylib
SPACK_DIR/htslib-1.16-su7mfvegt5xjb7o7lzulziwcigvpna64/lib/libhts.1.16.dylib
SPACK_DIR/htslib-1.16-su7mfvegt5xjb7o7lzulziwcigvpna64/lib/libhts.dylib
SPACK_DIR/htslib-1.16-su7mfvegt5xjb7o7lzulziwcigvpna64/lib/libhts.3.dylib
```